### PR TITLE
Some minor edits

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb [hunk 1]
```diff
@@ -78,32 +72,32 @@
...
+    "print(matplotlib.get_backend())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "which must be done *before* you `import matplotlib.pyplot as plt`.\n",
+    "\n",
+    "You can also set the backend via a 'magic ``%matplotlib backend_name``.  In addition to setting the backend, the magic also calls `plt.ion()`, which puts Matplotlib in 'interacitve mode' (the inverse is `plt.ioff()`).  In 'interactive mode' figures are shown (injected into the web page in the notebook) as soon as they are created.  Otherwise, figures are not shown until you explicitly call `plt.show()`.\n",
```
Comment (WeatherGod) [inline AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb]: missing closing quote for "magic". Also, we should probably make it clear that this is an "ipython magic".

File: AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb [hunk 2]
```diff
@@ -254,16 +239,15 @@
...
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "ax.set_xlim([0.5, 4.5])\n",
     "ax.set_ylim([-2, 8])\n",
-    "ax.set_title('An Example Axes')\n",
-    "ax.set_ylabel('Y-Axis')\n",
-    "ax.set_xlabel('X-Axis')"
+    "ax.set_title('An Diferent Example Axes Tile')\n",
```
Comment (WeatherGod) [inline AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb]: spelling

File: AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb [hunk 3]
```diff
@@ -254,16 +239,15 @@
...
+    "ax.set_ylabel('Y-Axis (changed)')\n",
+    "ax.set_xlabel('X-Axis (changed)')\n",
+    "fig.canvas.draw_idle()"
```
Comment (WeatherGod) [inline AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb]: what's the point of this draw_idle(). I really don't want this introduced so early.

File: AnatomyOfMatplotlib-Part3-HowToSpeakMPL.ipynb [hunk 4]
```diff
@@ -45,23 +43,26 @@
...
+    "\n",
+    "Matplotlib supports the [xkcd color names](https://xkcd.com/color/rgb/) prefxed by `'xkcd:'`.\n",
+    "\n",
+    "Matplotlib also understands  `{'tab:blue', 'tab:orange', 'tab:green', 'tab:red', 'tab:purple', 'tab:brown', 'tab:pink', 'tab:gray', 'tab:olive', 'tab:cyan'}` which are the Tableau Colors from the ‘T10’ categorical palette (and the default color cycle);\n",
     "\n",
     "### Hex values\n",
     "Colors can also be specified by supplying a HTML/CSS hex string, such as `'#0000FF'` for blue. Support for an optional alpha channel was added for v2.0.\n",
     "\n",
-    "### 256 Shades of Gray\n",
-    "A gray level can be given instead of a color by passing a string representation of a number between 0 and 1, inclusive. `'0.0'` is black, while `'1.0'` is white. `'0.75'` would be a light shade of gray.\n",
```
Comment (WeatherGod) [inline AnatomyOfMatplotlib-Part3-HowToSpeakMPL.ipynb]: Where did the explanation for grays go?

File: AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb [hunk 5]
```diff
@@ -1,20 +1,5 @@
...
-    "slideshow": {
-     "slide_type": "-"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "# Let printing work the same in Python 2 and 3\n",
-    "from __future__ import print_function"
-   ]
-  },
```
Comment (WeatherGod) [inline AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb]: I this this removal was unintentional?

Comment (tacaswell) [reply AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb]: No, I meant to strip it from all of them.

**Comment by WeatherGod** [comment]
Please don't. At least not until after the scipy conference. I did not
specify that the tutorial was py3k only.

**Comment by WeatherGod** [comment]
superseded by #20.

**Pull request closed without merging**